schema.graphql

```diff
@@ -27,6 +27,9 @@ type CollectionEntity @entity {
   blockNumber: BigInt
   meta: MetadataEntity
   createdAt: DateTime!
+  updatedAt: DateTime!
+  totalItems: Int!
+  totalAvailableItems: Int!
```

Comment:
```suggestion
  nftCount: Int!
  supply: Int!
```

Reply:
Can't commit your suggestion as we will clearly need to recreate a migration file
`nftCount` & `supply` feels meh :/ (from the perspective of a dev I find it hard to tell the difference) but I'll change that